src/main/java/de/dennisguse/opentracks/fragments/MarkerDetailFragment.java

```diff
@@ -188,6 +197,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
                         .putExtra(MarkerEditActivity.EXTRA_MARKER_ID, markerId);
                 startActivity(intent);
                 return true;
+            case R.id.marker_detail_share:
+                if (waypoint.hasPhoto()) {
+                    intent = IntentUtils.newShareImageIntent(getContext(), waypoint.getPhotoURI());
+                    Intent.createChooser(intent, null);
```

Comment:
needs to be:
intent = Intent.createChooser(intent, null);

Reply:
Ops, you are right. Sorry ;)

Reply:
Happens to me all the time :)